### PR TITLE
changing version in config

### PR DESCRIPTION
**Pull request by blackpanther1881**

### CODE REVIEW

File: src/containers/Wallet/Transactions/SendTransactions.js [hunk 1]
```diff
@@ -66,7 +66,11 @@ const SendTransactions = (props) => {
                 target="_blank" className="tx-hash" rel="noopener noreferrer">
                 {helper.stringTruncate(stxn.txhash)}
             </a>,
-            <span key={index} className="type">{(stxn.tx.body.messages[0]["@type"]).substr((stxn.tx.body.messages[0]["@type"]).indexOf('v1beta1.') + 11)}</span>,
+            (stxn.tx.body.messages[0]["@type"] === "/ibc.applications.transfer.v1.MsgTransfer") ?
+                <span key={index} className="type">{(stxn.tx.body.messages[0]["@type"]).substr((stxn.tx.body.messages[0]["@type"]).indexOf('/')+1)}</span>
```
Comment (puneet2019) [inline src/containers/Wallet/Transactions/SendTransactions.js:70]: are you sure indexing[0], 
will not cause any unexpected errors?

File: src/containers/Wallet/Transactions/SendTransactions.js [hunk 2]
```diff
@@ -66,7 +66,11 @@ const SendTransactions = (props) => {
...
+                :
```
Comment (puneet2019) [inline src/containers/Wallet/Transactions/SendTransactions.js:71]: what's this? +1, +11?

**Review by puneet2019 — APPROVED** [review]

File: src/containers/Wallet/Transactions/SendTransactions.js (hunk 1 above)
Comment (blackpanther1881) [reply src/containers/Wallet/Transactions/SendTransactions.js:70]: Yes, in index of 0 only getting type

**Review by deepanshutr — APPROVED** [review]
Reviewed.

**Pull request merged**